data/payloads.py

```diff
@@ -0,0 +1,15 @@
+def create_booking_payload(first_name, last_name, total_price, deposit_paid,
```

Comment:
gdy w metodzie pojawia się sporo parametrów warto pomyśleć o stworzeniu klasy, w tym przypadku to mógłby być `Booking` / `BookingDetails` / `BookingData`